dubbo-common/src/main/java/org/apache/dubbo/common/utils/CompatibleTypeUtils.java

```diff
@@ -119,8 +119,7 @@ public static Object compatibleTypeConvert(Object value, Class<?> type) {
                 } catch (ParseException e) {
                     throw new IllegalStateException(
                             "Failed to parse date " + value + " by format " + DATE_FORMAT + ", cause: "
-                                    + e.getMessage(),
-                            e);
```

Comment:
Revert this or will violate the style rules

Reply:
After repairing this area, it seems that it still doesn't work